zookeeper-server/src/test/java/org/apache/zookeeper/RemoveWatchesTest.java

```diff
@@ -82,15 +84,10 @@ public void tearDown() throws Exception {
         super.tearDown();
     }
 
-    private final boolean useAsync;
-
-    public RemoveWatchesTest(boolean useAsync) {
-        this.useAsync = useAsync;
-    }
-
-    @Parameters
-    public static Collection<Object[]> configs() {
-        return Arrays.asList(new Object[][]{{false}, {true}});
+    public static Stream<Arguments> data() throws Exception {
```

Comment:
for these trivial test parameter wouldn't be nicer to use `@ValueSource` instead of `@MethodSource` ?
see: https://www.baeldung.com/parameterized-tests-junit-5#1-simple-values

Reply:
the page above doesn't list `booleans` as annotation parameter, but according to the javadoc, it works too:  https://junit.org/junit5/docs/current/api/org.junit.jupiter.params/org/junit/jupiter/params/provider/ValueSource.html

Reply:
Fixed